### PR TITLE
Added Terraform files to deploy the basic counter example

### PR DESCRIPTION
**Pull request by icehaunter**
State is managed via GCS to work with CI in next PRs

### CODE REVIEW

File: demo_apps/basic_counter/main.tf [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  instance = var.cloudsql_database_instance.name
+  depends_on = [
+    var.cloudsql_database_instance
+  ]
+}
+
+resource "google_cloud_run_service" "demo" {
+  project = "vaxine"
+  name     = "basic-counter-demo"
+  location = "europe-west1"
```
Comment (thruflo) [inline demo_apps/basic_counter/main.tf:12]: The idea is to geo-distribute the service behind a global http load balancer, as per https://github.com/ahmetb/cloud-run-multi-region-terraform

**Review by thruflo — CHANGES_REQUESTED** [review]

File: demo_apps/basic_counter/main.tf (hunk 1 above)
Comment (icehaunter) [reply demo_apps/basic_counter/main.tf:12]: I though to leave that as a separate PR to be honest, since

Comment (thruflo) [reply demo_apps/basic_counter/main.tf:12]: Np to iterate in multiple PRs 👍

**Comment by define-null** [comment]
I suggest to add Makefile with some reasonable targets as well.

File: demo_apps/basic_counter/main.tf (hunk 1 above)
Comment (icehaunter) [reply demo_apps/basic_counter/main.tf]: 1. Added global scaling
2. https://basic-counter.examples.vaxine.io/

**Review by thruflo — APPROVED** [review]

**Pull request merged**